README.md

```diff
@@ -1,5 +1,5 @@
 # Instructions
 
 1. Clone this repository
-2. Checkout to a branch which has your name. Example: Deyan Dobrinov => `deyan-dobrinov`
-3. Solve questions [1; N] available in the root directory
+2. Create a new branch, use your first and last name. Example: Deyan Dobrinov => `deyan-dobrinov`
+3. Solve the `/questions` and add them in `/answers`
```

Comment:
The questions directory would contain a "pool" of questions. The idea is that I will be creating symbolic links in the root directory to a selected set of questions.